docs/getting_started/tfm_secure_boot.rst

```diff
@@ -254,6 +254,10 @@ modes are supported by which platforms:
 +---------------------+-----------------+---------------+----------+----------------+--------------+
 | STM_NUCLEO_L552ZE_Q | No              | Yes           | No       | No             | No           |
 +---------------------+-----------------+---------------+----------+----------------+--------------+
+| nRF5340             | No              | Yes           | No       | No             | No           |
```

Comment:
I think these are "DKs" not CPUs, so I would add PDK and DK, respectively.
E.g. NUCLEO is a board name, for STM32.